src/client/docs/import.md

```diff
@@ -0,0 +1,100 @@
+| drilling_method_id          | ID (Codeliste) | Nein        | Bohrmethode                               |
+| purpose_id                  | ID (Codeliste) | Nein        | Bohrzweck                                 |
+| spud_date                   | Datum          | Nein        | Datum Bohrbeginn                          |
+| cuttings_id                 | ID (Codeliste) | Nein        | Bohrgut                                   |
+| status_id                   | ID (Codeliste) | Nein        | Bohrungsstatus                            |
+| inclination                 | Dezimalzahl    | Nein        | Inklination beim Bohransatzpunkt [°]      |
+| inclination_direction       | Dezimalzahl    | Nein        | Einfallsrichtung beim Bohransatzpunkt [°] |
+| qt_inclination_direction_id | ID (Codeliste) | Nein        | Inc. / Richtung Genauigkeit               |
+| remarks                     | Text           | Nein        | Bemerkungen                               |
+| total_depth                 | Dezimalzahl    | Nein        | Bohrlochlänge [m MD]                      |
+| qt_depth_id                 | ID (Codeliste) | Nein        | +/- Bohrlochlänge [m MD]                  |
+| total_depth_tvd             | Dezimalzahl    | Nein        | Vertikale Bohrendteufe [m TVD]            |
+| qt_total_depth_tvd_id       | ID (Codeliste) | Nein        | +/- Vertikale Bohrendteufe [m TVD]        |
+| top_bedrock                 | Dezimalzahl    | Nein        | Top Fels [m MD]                           |
+| qt_top_bedrock_id           | ID (Codeliste) | Nein        | +/- Top Fels [m MD]                       |
+| top_bedrock_tvd             | Dezimalzahl    | Nein        | Top Fels [m TVD]                          |
+| qt_top_bedrock_tvd_id       | ID (Codeliste) | Nein        | +/- Top Fels [m TVD]                      |
+| has_groundwater             | True/False     | Nein        | Grundwasser                               |
+| lithology_top_bedrock_id    | ID (Codeliste) | Nein        | Lithologie Top Fels                       |
+| chronostratigraphy_id       | ID (Codeliste) | Nein        | Chronostratigraphie Top Fels              |
+| lithostratigraphy_id        | ID (Codeliste) | Nein        | Lithostratigraphie Top Fels               |
+
+## Validierung
+
+Beim Import der CSV-Datei wird der Importprozess die Bohrdaten validieren und mögliche Duplikate identifizieren. Die Duplikaterkennung erfolgt anhand der Koordinaten (mit einer Toleranz von +/- 2 Metern) und der Gesamttiefe des Bohrlochs. Wenn ein Duplikat entdeckt wird, wird der Import abgebrochen und es wird eine entsprechende Fehlermeldung angezeigt.
+
+Während der Validierung werden auch die zugehörigen Anhänge überprüft. Wenn ein Anhang in der CSV-Datei referenziert wird, aber nicht in den bereitgestellten Anhangsdateien gefunden wird, wird ebenfalls eine Fehlermeldung angezeigt und der Import abgebrochen.
```

Comment:
Q: Wie wird ein Anhang oder mehrere Anhänge in der CSV Datei für eine Bohrung referenziert? Was ist mit Anhängen, welche für mehrere Bohrungen verwendet werden? Was ist wenn ein Anhang bereits im System ist, muss oder soll der Anhang dann nochmals referenziert werden, wie?

Reply:
Diese Doku wird noch nachgeliefert und fehlt momentan. Das hängt mit den Arbeiten von @Lehats zusammen und wird durch ihn dokumentiert.